app/controllers/countries_controller.rb

```diff
@@ -3,4 +3,16 @@ def show
     @country = Country.find(params[:id])
     @cities = City.where(country_id: params[:id]).order("name")
   end
+
+  def index
+    if params[:q]
+      if params[:q] == ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.